e2e/helm/helm.go

```diff
@@ -38,10 +35,9 @@ func InstallRelease(path, context string, logger *logrus.Logger, config ...strin
 	c := exec.Command(_helmBinary, args...)
 	o, err := c.CombinedOutput()
 	if err != nil {
-		return nil, fmt.Errorf("%s - %s", err, string(o))
+		return fmt.Errorf("%s - %s", err, string(o))
```

Comment:
```suggestion
		return fmt.Errorf("%w - %s", err, string(o))
```

Reply:
I updated also other occurrences in the code